src/targets/gpu/CMakeLists.txt

```diff
@@ -332,6 +332,7 @@ add_subdirectory(driver)
 add_subdirectory(hiprtc)
 
 rocm_install_targets(
+    PRIVATE
```

Comment:
GPU lib has binaries for `hiprtc-driver` and also for `gpu-driver` i guess. I think they would need to be marked as private as well.

Reply:
Yea for `hiprtc-driver`, it could be marked as `PRIVATE`, but we can update that in a later PR as we would need to update how we search for the binary as well.

Reply:
@ahsan-ca  is there an open issue for moving hiprtc-driver and gpu-driver ? if not can you open one and work on it as next part ?

Reply:
Sure, I will open up an issue for this.